src/examples/mc_pos_control_multiplatform/mc_pos_control.cpp

```diff
@@ -640,8 +640,10 @@ void  MulticopterPositionControlMultiplatform::handle_vehicle_attitude(const px4
 		    && _pos_sp_triplet->data().current.type == _pos_sp_triplet->data().current.SETPOINT_TYPE_IDLE) {
 			/* idle state, don't run controller and set zero thrust */
 			_R.identity();
-			memcpy(&_att_sp_msg.data().R_body[0], _R.data, sizeof(_att_sp_msg.data().R_body));
-			_att_sp_msg.data().R_valid = true;
+			math::Quaternion qd;
+			qd.from_dcm(_R);
+			memcpy(&_att_sp_msg.data().q_d[0], qd.data, sizeof(_att_sp_msg.data().q_d));
```

Comment:
Syntax cleanup: here  reference operator (&) used, but not necessary. suggestion:
memcpy(_att_sp_msg.data().q_d, qd.data, sizeof(_att_sp_msg.data().q_d))